### DISCUSSION TITLE
Chunking question

### DISCUSSION THREAD

**Discussion by E2GO** [lllyasviel/Fooocus#3299] [Q&A] [answered]
I've stumbled upon several Reddit posts where people mention using the BREAK command (chunking) in prompts for Fooocus, similar to how it's often used in A1111. However, as far as I know, Fooocus doesn't have this functionality. Or has it been implemented somehow already?

Just to clarify, I'm referring to this feature: https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Features#break-keyword

**Comment by mashb1t** [comment, marked as answer]
Fooocus does not support the BREAK keyword, but using multiple lines serves a similar purpose. Keep in mind that only the first line supports inline wildcards / LoRAs.
